decidim-core/db/migrate/20220119151540_add_allow_push_notifications_to_users.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddAllowPushNotificationsToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :decidim_users, :allow_push_notifications, :boolean, default: false
```

Comment:
After #8757 we have the `notifications_settings` JSONB column.

I see that's actually being used by the related PR #8774, so I think it makes sense to also migrate this column to `notifications_settings`, so it's all consistent. I know it can be a bit more of an effort but in the long term it's going to be there, so I think it's better to do it sooner than later. What do you think @quinHD?

Reply:
since this column will disappear in the other PR. We have considered that this PR can be closed and its functionality will be taken over by the [push-notifications-to-client ](https://github.com/decidim/decidim/pull/8774)PR.